drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -8392,8 +8415,88 @@ void RasterizerStorageGLES3::initialize() {
 		config.anisotropic_level = MIN(int(ProjectSettings::get_singleton()->get("rendering/quality/filters/anisotropic_filter_level")), config.anisotropic_level);
 	}
 
+#ifdef GLES_OVER_GL
+	config.program_binary_supported = GLAD_GL_ARB_get_program_binary;
+	config.parallel_shader_compile_supported = GLAD_GL_ARB_parallel_shader_compile || GLAD_GL_KHR_parallel_shader_compile;
+#else
+	config.program_binary_supported = true;
```

Comment:
We are going to need a special case for WebGL as it never supports glProgramBinary

Reply:
Good point. I guess that in the WebGL case we will only have the possibility of using the approach based on the parallel compile extension, with no fallback. Also, caching won't be possible at all.

Reply:
Unfortunately yeah. :(